File: app/src/main/java/com/example/myapp/WeatherData/Coord.java
package com.example.myapp.WeatherData;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Coord {
    @SerializedName("lat")
    @Expose
    public float lat;
    @SerializedName("lon")
    @Expose
    public float lon;
}

File: app/src/main/java/com/chplalex/shaman/mvp/model/Coord.kt
package com.chplalex.shaman.mvp.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Coord(
    @SerializedName("lat") @Expose
    var lat: Float = 0f,
    @SerializedName("lon") @Expose
    var lon: Float = 0f
)